### PR TITLE
#1905 Using directive should follow OSS generics coding standard

### LINKED ISSUE #1905: using Directive should follow OSS-Generics coding standard
Repeat of old issue #1533.

### PR DESCRIPTION
**Pull request by yuhongherald**
Fixes #1905

### CODE REVIEW

File: PowerPointLabs/Test/UnitTest/ELearningLab/Model/ClickItemTest.cs [hunk 1]
```diff
@@ -1,9 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
+﻿
```
Comment (YuPeiHenry) [inline PowerPointLabs/Test/UnitTest/ELearningLab/Model/ClickItemTest.cs]: Leading whitespace

File: PowerPointLabs/Test/UnitTest/ELearningLab/Model/ComputerVoiceTest.cs [hunk 2]
```diff
@@ -1,10 +1,6 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
+﻿
```
Comment (YuPeiHenry) [inline PowerPointLabs/Test/UnitTest/ELearningLab/Model/ComputerVoiceTest.cs]: Leading whitespace

File: PowerPointLabs/Test/UnitTest/ELearningLab/Model/SelfExplanationClickItemTest.cs [hunk 3]
```diff
@@ -1,9 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
+﻿
```
Comment (YuPeiHenry) [inline PowerPointLabs/Test/UnitTest/ELearningLab/Model/SelfExplanationClickItemTest.cs]: Leading whitespace

**Comment by yuhongherald** [comment]
I have removed the leading newlines.

**Comment by YuPeiHenry** [comment]
@yuhongherald Can you resolve the merge conflict? Thanks.

**Pull request merged**